<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="2" ignored="0" retried="2" total="4" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="91143" started-at="2020-08-10T20:02:37 GMT+05:30" finished-at="2020-08-10T20:04:08 GMT+05:30">
    <groups>
      <group name="WeatherreportingTestcaess">
        <method signature="NewTest.VerifyHumidity()[pri:1, instance:testcases.NewTest@2c039ac6]" name="VerifyHumidity" class="testcases.NewTest"/>
        <method signature="NewTest.VerifyTemperature()[pri:2, instance:testcases.NewTest@2c039ac6]" name="VerifyTemperature" class="testcases.NewTest"/>
      </group> <!-- WeatherreportingTestcaess -->
    </groups>
    <test name="Default test" duration-ms="91143" started-at="2020-08-10T20:02:37 GMT+05:30" finished-at="2020-08-10T20:04:08 GMT+05:30">
      <class name="testcases.NewTest">
        <test-method status="PASS" signature="configForWebpageLaunch()[pri:0, instance:testcases.NewTest@2c039ac6]" name="configForWebpageLaunch" is-config="true" duration-ms="12610" started-at="2020-08-10T20:02:37 GMT+05:30" finished-at="2020-08-10T20:02:49 GMT+05:30">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- configForWebpageLaunch -->
        <test-method status="SKIP" signature="VerifyHumidity()[pri:1, instance:testcases.NewTest@2c039ac6]" name="VerifyHumidity" duration-ms="3693" retried="true" started-at="2020-08-10T20:02:49 GMT+05:30" finished-at="2020-08-10T20:02:53 GMT+05:30">
          <exception class="org.json.JSONException">
            <message>
              <![CDATA[JSONObject["current"] not found.]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.json.JSONException: JSONObject["current"] not found.
at org.json.JSONObject.get(JSONObject.java:572)
at org.json.JSONObject.getJSONObject(JSONObject.java:782)
at testcases.NewTest.VerifyHumidity(NewTest.java:41)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.json.JSONException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- VerifyHumidity -->
        <test-method status="PASS" signature="tearDownTest()[pri:0, instance:testcases.NewTest@2c039ac6]" name="tearDownTest" is-config="true" duration-ms="5816" started-at="2020-08-10T20:02:53 GMT+05:30" finished-at="2020-08-10T20:02:59 GMT+05:30">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownTest -->
        <test-method status="PASS" signature="configForWebpageLaunch()[pri:0, instance:testcases.NewTest@2c039ac6]" name="configForWebpageLaunch" is-config="true" duration-ms="13709" started-at="2020-08-10T20:02:59 GMT+05:30" finished-at="2020-08-10T20:03:13 GMT+05:30">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- configForWebpageLaunch -->
        <test-method status="FAIL" signature="VerifyHumidity()[pri:1, instance:testcases.NewTest@2c039ac6]" name="VerifyHumidity" duration-ms="1098" started-at="2020-08-10T20:03:13 GMT+05:30" finished-at="2020-08-10T20:03:14 GMT+05:30">
          <exception class="org.json.JSONException">
            <message>
              <![CDATA[JSONObject["current"] not found.]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.json.JSONException: JSONObject["current"] not found.
at org.json.JSONObject.get(JSONObject.java:572)
at org.json.JSONObject.getJSONObject(JSONObject.java:782)
at testcases.NewTest.VerifyHumidity(NewTest.java:41)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.retryFailed(TestInvoker.java:204)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:58)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.json.JSONException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- VerifyHumidity -->
        <test-method status="PASS" signature="tearDownTest()[pri:0, instance:testcases.NewTest@2c039ac6]" name="tearDownTest" is-config="true" duration-ms="5844" started-at="2020-08-10T20:03:14 GMT+05:30" finished-at="2020-08-10T20:03:20 GMT+05:30">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownTest -->
        <test-method status="PASS" signature="configForWebpageLaunch()[pri:0, instance:testcases.NewTest@2c039ac6]" name="configForWebpageLaunch" is-config="true" duration-ms="23392" started-at="2020-08-10T20:03:20 GMT+05:30" finished-at="2020-08-10T20:03:43 GMT+05:30">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- configForWebpageLaunch -->
        <test-method status="SKIP" signature="VerifyTemperature()[pri:2, instance:testcases.NewTest@2c039ac6]" name="VerifyTemperature" duration-ms="1165" retried="true" started-at="2020-08-10T20:03:43 GMT+05:30" finished-at="2020-08-10T20:03:44 GMT+05:30">
          <exception class="org.json.JSONException">
            <message>
              <![CDATA[JSONObject["current"] not found.]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.json.JSONException: JSONObject["current"] not found.
at org.json.JSONObject.get(JSONObject.java:572)
at org.json.JSONObject.getJSONObject(JSONObject.java:782)
at testcases.NewTest.VerifyTemperature(NewTest.java:58)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.json.JSONException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- VerifyTemperature -->
        <test-method status="PASS" signature="tearDownTest()[pri:0, instance:testcases.NewTest@2c039ac6]" name="tearDownTest" is-config="true" duration-ms="5693" started-at="2020-08-10T20:03:44 GMT+05:30" finished-at="2020-08-10T20:03:50 GMT+05:30">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownTest -->
        <test-method status="PASS" signature="configForWebpageLaunch()[pri:0, instance:testcases.NewTest@2c039ac6]" name="configForWebpageLaunch" is-config="true" duration-ms="10420" started-at="2020-08-10T20:03:50 GMT+05:30" finished-at="2020-08-10T20:04:00 GMT+05:30">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- configForWebpageLaunch -->
        <test-method status="FAIL" signature="VerifyTemperature()[pri:2, instance:testcases.NewTest@2c039ac6]" name="VerifyTemperature" duration-ms="1343" started-at="2020-08-10T20:04:00 GMT+05:30" finished-at="2020-08-10T20:04:02 GMT+05:30">
          <exception class="org.json.JSONException">
            <message>
              <![CDATA[JSONObject["current"] not found.]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.json.JSONException: JSONObject["current"] not found.
at org.json.JSONObject.get(JSONObject.java:572)
at org.json.JSONObject.getJSONObject(JSONObject.java:782)
at testcases.NewTest.VerifyTemperature(NewTest.java:58)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.retryFailed(TestInvoker.java:204)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:58)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.json.JSONException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- VerifyTemperature -->
        <test-method status="PASS" signature="tearDownTest()[pri:0, instance:testcases.NewTest@2c039ac6]" name="tearDownTest" is-config="true" duration-ms="6297" started-at="2020-08-10T20:04:02 GMT+05:30" finished-at="2020-08-10T20:04:08 GMT+05:30">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownTest -->
      </class> <!-- testcases.NewTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
